src/App.tsx

```diff
@@ -15,36 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type Params = {
+  sortfield: SortType;
```

Comment:
The property name `sortfield` should be `sortField` to maintain consistency with the camelCase naming convention used throughout the code.

Reply:
> The property name `sortfield` should be `sortField` to maintain consistency with the camelCase naming convention used throughout the code.

Good point